### DISCUSSION TITLE
More Draw Element Stroke Width Options

### DISCUSSION THREAD

**Discussion by igneous8k** [excalidraw/excalidraw#5899] [Q&A]
HI, im trying to figure out to extend stroke width in draw element, like there are 3 options available i would also wanna add + and - button to increase even further. I'd appreciate any help.
im looking for mainly stroke width but i also wanna know few things also which are :
1.) draw pen custom brush like custom shaped brush to draw .
2.) Stroke Width using increment and decrement buttons. (Beyond 3 available units ).

:)

**Comment by 200found** [comment]
i would also really enjoy larger brush sizes and custom brushes.
